docs/paper/paper.md

```diff
@@ -0,0 +1,88 @@
+---
+title: TorchMetrics - Measuring Reproducibility in PyTorch
+tags:
+  - python
+  - deep learning
+  - pytorch
+authors:
+  - Nicki Skafte Detlefsen
+    affiliation: "1,2"
+  - Jirka Borovec
+    affiliation: "1"
+  - Justus Schock
+    affiliation: "1"
+  - Ananya Harsh Jha
+    affiliation: "1"
+  - Teddy Koker
+    affiliation: "1"
+  - Lucadiliello
+  - Stancld
```

Comment:
@Stancld ^^

Reply:
```suggestion
  - Daniel Stancl
```
affiliation: Charles University

Reply:
mind rereview after the update? :rabbit: